smartmontools.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+STRAPPROJECTS += smartmontools 
```

Comment:
This should be a ``SUBPROJECT``, not a ``STRAPPROJECT``. Fix your var spacing as well.